homeassistant/components/playstation_network/media_player.py

```diff
@@ -0,0 +1,139 @@
+"""Media player entity for the Playstation Network Integration."""
+
+from dataclasses import dataclass
+import logging
+
+from homeassistant.components.media_player import (
+    MediaPlayerDeviceClass,
+    MediaPlayerEntity,
+    MediaPlayerState,
+    MediaType,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import PlaystationNetworkCoordinator
+from .coordinator import PlaystationNetworkData
+from .entity import PlaystationNetworkEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class PSNdata:
+    """PSN dataclass."""
+
+    account = {"id": "", "handle": ""}
+    presence = {"availability": "", "lastAvailableDate": ""}
+    platform = {"status": "", "platform": ""}
+    title = {"name": "", "format": "", "imageURL": None, "playing": False}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Media Player Entity Setup."""
+    coordinator: PlaystationNetworkCoordinator = config_entry.runtime_data
+
+    if coordinator.data.platform.get("platform") is None:
+        username = coordinator.data.username
+        _LOGGER.warning(
+            "No console found associated with account: %s. -- Skipping creation of media player",
+            username,
+        )
+        return
+
+    async_add_entities([MediaPlayer(coordinator)])
+
+
+class MediaPlayer(PlaystationNetworkEntity, MediaPlayerEntity):
+    """Media player entity representing currently playing game."""
+
+    device_class = MediaPlayerDeviceClass.TV
+
+    def __init__(self, coordinator: PlaystationNetworkCoordinator) -> None:
+        """Initialize PSN MediaPlayer."""
+        super().__init__(coordinator)
+        self.psn: PlaystationNetworkData = self.coordinator.data
+        self._attr_has_entity_name = True
```

Comment:
```suggestion
        self._attr_has_entity_name = True
```
this would be obsolete if you add an entity_description